### ISSUE TITLE
Loss Function

### ISSUE THREAD

**Issue by YingdiZhang** [jacobgil/keras-dcgan#20]
In the paper GAN,the loss funtion has a form of minmaxV(G,D),while in your code the loss is 'binary_crossentropy',I'm little confused about this,could you please explain this a little bit?
